### PR TITLE
`get_relative_path, strip_relative_path, get_asset_url`

### PR DESCRIPTION
**Pull request by AnnMarieW**
Closes https://github.com/plotly/dash-labs/issues/65

This PR creates the following functions:
- `dash.get_relative_path`
- `dash.strip_relative_path`
- `dash.get_asset_url`

This is similar to `dash.callback` where you don't need the `app` object. It makes it possible to use these functions in the `pages` folder of a multi-page app without running into the circular `app` imports issue.

### CODE REVIEW

File: dash/dash.py [hunk 1]
```diff
@@ -424,6 +424,9 @@ def __init__(
 
         self.logger.setLevel(logging.INFO)
 
+    def _get_config(self):
+        return self.config
```
Comment (alexcjohnson) [inline dash/dash.py:428]: is this needed?

Comment (AnnMarieW) [reply dash/dash.py:428]: Nope - that was an early attempt at a solution.  I'll delete it

Comment (AnnMarieW) [reply dash/dash.py]: done

**Review by alexcjohnson — APPROVED** [review]
💃 very nice!

**Pull request merged**